### ISSUE TITLE
Too many npm scripts

### ISSUE THREAD

**Issue by yucho** [catscradleio/catscradle.io#39]
@demetrios711 @kvebox @ndo277 Guys, currently we got too many npm scripts. Let's remove some useless scripts! Right now we got:
```JSON
"scripts": {
    "build": "webpack --mode=production",
    "dev": "concurrently \"npm run server\" \"npm run frontend\"",
    "develop": "webpack --mode=development",
    "frontend": "npm start --prefix frontend",
    "frontend-install": "npm install --prefix frontend",
    "postinstall": "cd frontend && npm install",
    "server": "nodemon --exec babel-node backend/server.js",
    "start": "node backend/server.js"
  }
```

I'd say remove `build, develop, frontend-install, start` because they're not being used, and rename `server` -> `backend`, and `dev` -> `server`

**Comment by demetrios711** [comment]
I support this change.

**Comment by yucho** [comment]
Closed by #44
